### PR TITLE
Add verbosity as argument for CLI

### PR DESCRIPTION
**Pull request by Virviil**
New CLI arg was added - `-v` | `--verbose`, that is more-less standard for CLIs, that allows to trace problems running CLI.
Before the PR one can do it just by setting `RUST_LOG` env, which is poorly documented.

### CODE REVIEW

File: loco-cli/src/bin/main.rs [hunk 1]
```diff
@@ -24,8 +29,21 @@ enum Commands {
 fn main() -> eyre::Result<()> {
     let cli = Cli::parse();
 
+    println!("{:?}", &cli.verbose);
```
Comment (jondot) [inline loco-cli/src/bin/main.rs:32]: you probably want to remove this right?

Comment (Virviil) [reply loco-cli/src/bin/main.rs:32]: Y, sure. One more question - will this change break a test? Probably I should fix it also

Comment (kaplanelad) [reply loco-cli/src/bin/main.rs:32]: @Virviil, you are correct. See [here](https://github.com/loco-rs/loco/blob/master/loco-cli/tests/cli_tests.rs#L3)  it is commented out. 
Do you mind to fix it?

Comment (Virviil) [reply loco-cli/src/bin/main.rs]: Fixed

Comment (Virviil) [reply loco-cli/src/bin/main.rs]: @kaplanelad I'm not sure that i'll take this task, but for sure not as a part of this PR. So let's just proceed

**Review by kaplanelad — APPROVED** [review]

**Pull request merged**